server/jetstream_test.go

```diff
@@ -1853,13 +1853,15 @@ func TestJetStreamWorkQueueRequest(t *testing.T) {
 			// Send a few more messages. These should not be delivered to the sub.
 			sendStreamMsg(t, nc, "foo", "Hello World!")
 			sendStreamMsg(t, nc, "bar", "Hello World!")
-			// We will have an alert here.
+			time.Sleep(10 * time.Millisecond)
```

Comment:
Would make at least 100 these days.

Reply:
We sleep above past the expiration time. This sleep was to make sure that we would not start delivering, but I hear you. Will increase.

Reply:
Increased the expiration timeout anyway (but still "small") and bumped server version.